composer.json

```diff
@@ -38,12 +38,12 @@
         "symfony/console": "^4.0|^5.0"
     },
     "require-dev": {
-        "behat/behat": "^3.7",
-        "phpspec/phpspec": "^6.1",
-        "friends-of-phpspec/phpspec-code-coverage": "^4.3",
-        "friendsofphp/php-cs-fixer": "^2.16",
-        "moave/phpspec-data-provider-extension": "dev-master",
-        "vimeo/psalm": "^3.12"
+        "behat/behat": "^3.8",
```

Comment:
It's so important this change? Behat support php8 from 3.8version?

Reply:
Exactly: https://packagist.org/packages/behat/behat#v3.8.0

Reply:
ok perfect 👍